examples/typescript/http-router/spin.toml

```
@@ -1,15 +1,17 @@
-spin_manifest_version = "1"
+spin_manifest_version = 2
+
+[application]
 authors = ["Thorsten Hans <thorsten.hans@gmail.com>"]
 description = "Sample to demonstrate the usage of theHTTP router"
```

dunno if you're fixing typos but 
```suggestion
description = "Sample to demonstrate the usage of the HTTP router"
```